packages/shared/lib/contexts/onboarding/actions/newProfile.ts

```diff
@@ -1,10 +1,10 @@
 import { destroyProfileManager, profileManager } from '@core/profile-manager'
```

Comment:
Can we separate these two functions into their own file and add them to the index. Then where we use them we can we also change it to just import from contexts/onboarding?